yarn.lock

```diff
@@ -14799,6 +14897,15 @@ ts-loader@^8.0.17:
     micromatch "^4.0.0"
     semver "^7.3.4"
 
+<<<<<<< HEAD
```

Comment:
Yarn has a nice feature that it merges lock files automatically. So my strategy is usually the following: pull changes, get warning about merge conflicts, fix merge conflicts to files myself and run `yarn install` which takes care of the conflicts in the lock file, then commit merge.

Reply:
I don't feel like this is a good soln. Is there any other better way to get Redis Client?